frame/clearing-house/src/types.rs

```diff
@@ -72,6 +103,24 @@ pub struct Market<T: Config> {
 	/// Minimum amount of quote asset to exchange when opening a position. Also serves to round
 	/// a trade if it results in closing an existing position.
 	pub minimum_trade_size: T::Decimal,
+	/// The time span between each funding rate update.
+	pub funding_frequency: DurationSeconds,
+	/// Period of time over which funding (the difference between mark and index prices) gets
+	/// paid.
+	///
+	/// Setting the funding period too long may cause the perpetual to start trading at a
+	/// very dislocated price to the index because there’s less of an incentive for basis
```

Comment:
Extremely nitpick:

For consistency sake change `’` to `'` in `... there's ...`